py-polars/polars/dataframe/frame.py

```diff
@@ -2309,6 +2308,7 @@ def write_json(
         *,
         pretty: bool = ...,
         row_oriented: bool = ...,
+        storage_options: dict[str, Any] | None = None,
```

Comment:
Entries in overload signature should not define default values: 
```suggestion
        storage_options: dict[str, Any] | None = ...,
```

Some goes for a lot of other spots.